cmd/recordtester/recordtester.go

```diff
@@ -419,7 +420,7 @@ func main() {
 					TesterOptions:           ttOpts,
 				}
 				ctt := transcodetester.NewContinuousTranscodeTester(egCtx, cttOpts)
-				return ctt.Start(*fileArg, *transcodeBucketUrl, *testDuration, *taskPollDuration, *continuousTest)
+				return ctt.Start(*fileArg, *transcodeBucketUrl, *transcodeW3sProof, *testDuration, *taskPollDuration, *continuousTest)
```

Comment:
I'd say the ideal place for most new config should be in the `cttOpts` instead (likely in the underlying `ttOpts`).

Honestly I don't really know why some of these fields were added in the `Start` function instead of options of the tester, but adding a field to the struct is much easier to do and maintain anyway.

Reply:
I think the reason is that `cttOpts` (and `ttOpts`) are shared options between different testers (vod and transcode) and everything we put inside `Start()` is specific to the given test.

Saying that, I agree it's better to have this in some kind of structs. I'll refactor it accordingly.

Reply:
ok, I started the refactor, but then I realized that to refactor it properly it's rather a day or two, rather than an hour or two. So, I created the related GH Issue.

I'm not sure we want to spend time on that right now.

Here's the GH Issue: https://github.com/livepeer/stream-tester/issues/268
Here's the started refactor: https://github.com/livepeer/stream-tester/tree/rafal/refactor-recordtester